src/net.cpp

```diff
@@ -1223,41 +1223,40 @@ void CConnman::DisconnectNodes() {
 }
 
 void CConnman::InactivityChecks() {
-    LOCK(cs_vNodes);
-    for (CNode* pnode : vNodes) {
-        InactivityCheck(pnode);
-    }
-}
-
-void CConnman::InactivityCheck(CNode *pnode) {
     int64_t nMicroTime = GetTimeMicros();
     int64_t nTime = nMicroTime / 1000000;
-    if (nTime - pnode->nTimeConnected > 60)
+
     {
```

Comment:
Commit "Merge InactivityCheck into InactivityChecks."

Could remove this block?

Reply:
Not sure what you mean here

Reply:
I mean like this:
```cpp
void CConnman::InactivityChecks() {
    LOCK(cs_vNodes);
    int64_t nMicroTime = GetTimeMicros();
    int64_t nTime = nMicroTime / 1000000;
    for (CNode* pnode : vNodes) {
        if (nTime - pnode->nTimeConnected > 60)
        ...
```

Reply:
yeah good point, the lock could take a while